### DISCUSSION TITLE
swap partition with ssd nvme m2

### DISCUSSION THREAD

**Discussion by sajjad000** [linuxdeepin/developer-center#3500] [Q&A | 问答板块]
Does swap partition affect the lifespan of an nvme m2 ssd?

**Comment by BLumia** [comment]
Well, technically yes since the data was actually getting write to your SSD while using...

**Reply by sajjad000** [reply]
I have 8 GB RAM, do I avoid creating swap partition

**Reply by BLumia** [reply]
Depends on your usage of the computer, *If* you memory usage will larger than your RAM in most of the cases, then you may consider using a swap partition (or swap file), but normally you no longer need a swap pertition IMO :)
